utils/constants.lua

```diff
@@ -0,0 +1,26 @@
+local M = {}
+
+M.ABILITY_LIST = {"STR", "DEX", "CON", "INT", "WIS", "CHA"}
+
+M.SR_LIST = {"1/8", "1/4", "1/2", "1", "2", "3", "4", "5", "6", "7","8", "9", "10", "11", "12", "13", "14", "15"}
+M.SR_TO_NUMBER = {["1/8"]=0.125, ["1/4"]=0.25, ["1/2"]=0.5, ["1"]=1, ["2"]=2, ["3"]=3, ["4"]=4, ["5"]=5, ["6"]=6, ["7"]=7,["8"]=8, ["9"]=9, ["10"]=10, ["11"]=11, ["12"]=12, ["13"]=13, ["14"]=14, ["15"]=15}
```

Comment:
Couldn't you set __index for this and avoid needing to define everything 1+?

Reply:
It's currently used to get the lowest and highest SR, in that case `__index` would do it. I could iterate over it and then add 1-15 to it like that.